everviz/plugins/_crossplot_plugin/crossplot_control_group_plugin.py

```diff
@@ -0,0 +1,161 @@
+            self.radio_items_y_id,
+        )
+        realizations_dropdowns = [
+            html.Div(
+                [
+                    dcc.RadioItems(
+                        id=self.interpolation_id,
+                        options=[
+                            {"label": label, "value": value}
+                            for label, value in [
+                                ("None", None),
+                                ("Linear", "linear"),
+                                ("High value", "hv"),
+                                ("Low value", "vh"),
+                            ]
+                        ],
+                        value=None,
+                        labelStyle={"display": "inline-block"},
+                    ),
+                    "Realization 1",
+                    dcc.Dropdown(
+                        id=self.dropdown_realization_id,
+                        options=[{"label": i, "value": i} for i in dropdown_options],
+                        placeholder="Select a realization",
+                    ),
+                    "Realization 2",
+                    dcc.Dropdown(
+                        id=self.dropdown_realization_2_id,
+                        options=[{"label": i, "value": i} for i in dropdown_options],
+                        placeholder="Select a realization",
+                    ),
+                ],
+                style={"width": "60%", "display": "inline-block"},
+            )
+        ]
+
+        return html.Div(
+            basic_layout + realizations_dropdowns, style={"horizontal-align": "middle"}
+        )
+
+    def set_callbacks(self, app):
+        @app.callback(
+            Output(self.graph_id, "figure"),
+            [
+                Input(self.dropdown_x_id, "value"),
+                Input(self.dropdown_y_id, "value"),
+                Input(self.dropdown_realization_id, "value"),
+                Input(self.radio_items_x_id, "value"),
+                Input(self.radio_items_y_id, "value"),
+                Input(self.dropdown_realization_2_id, "value"),
+                Input(self.interpolation_id, "value"),
+            ],
+        )
+        def update_graph(
+            xaxis_name,
+            yaxis_name,
+            realization_nr_1,
+            xaxis_type,
+            yaxis_type,
+            realization_nr_2,
+            interpolation_type,
+        ):
+            df = get_data(self.data_path)
+            trace_1 = _get_trace(
+                df,
+                xaxis_name,
+                yaxis_name,
+                realization_nr_1,
+                xaxis_type,
+                yaxis_type,
+                interpolation_type,
+            )
+            trace_2 = _get_trace(
+                df,
+                xaxis_name,
+                yaxis_name,
+                realization_nr_2,
+                xaxis_type,
+                yaxis_type,
+                interpolation_type,
+            )
+            data = [trace_1, trace_2]
```

Comment:
Suppose you would want this in a list comprehension, only one variable changing.